packages/odyssey-react-mui/src/MenuButton.tsx

```diff
@@ -65,10 +82,25 @@ export type MenuButtonProps = {
    * If the MenuButton is an overflow menu or standard menu.
    */
   isOverflow?: boolean;
+  /**
+   * The horizontal alignment of the menu.
+   */
+  menuAlignment?: "left" | "right";
```

Comment:
Don't we want this to be an array of values that we set as a type and pull into Storybook?